src/Goteo/Library/Forms/Admin/AdminFootprintEditForm.php

```diff
@@ -110,14 +106,26 @@ public function save(FormInterface $form = null, $force_save = false) {
 
         $data = $form->getData();
         // Dropfiles type always return an array, just get the first element if required
-        if($data['icon'] && is_array($data['icon'])) {
-            $data['icon'] = $data['icon'][0];
-        } else {
-            $data['icon'] = null;
-        }
+        // if($data['icon'] && is_array($data['icon'])) {
+        //     $data['icon'] = $data['icon'][0];
+        // } else {
+        //     $data['icon'] = null;
+        // }
```

Comment:
@davidbeig what do you think if we get rid of this commented code? We already have code versioning to keep history of it :stuck_out_tongue: